code/API_definitions/location-verification.yaml

```diff
@@ -12,7 +12,7 @@ info:
     
     * If the network's estimation of the device's location is fully contained within the requested area, the verification result is `TRUE`.
     * If the network's estimation of the device's location does not overlap with the requested area at all, the verification result is `FALSE` . 
-    * If the network's estimation of the device's location partially overlaps with the requested area, or it fully contains the requested area (because it is larger), the result is 'PARTIAL'. In this case, a `match_rate` may be included in the response, indicating an estimation of the likelihood of the match in percent.
+    * If the network's estimation of the device's location partially overlaps with the requested area, or it fully contains the requested area (because it is larger), the result is 'PARTIAL'. In this case, a `match_rate` is included in the response, indicating an estimation of the likelihood of the match in percent.
```

Comment:
By saying "may be included", we allow the implementer _not to include_ the matchRate even if the match is partial.
If we remove "may be", we make the matchRate _effectively_ mandatory if match is partial.
Is that what we want to do ?

Reply:
What would be the advantage of having just "PARTIAL" without a match_rate?

Reply:
Not really an advantage, just a convenience for the implementer not to implement the calculation part.